third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -355,11 +405,10 @@ func flattenSecurityPolicyRules(rules []*compute.SecurityPolicyRule) []map[strin
 			"match": []map[string]interface{}{
 				{
 					"versioned_expr": rule.Match.VersionedExpr,
-					"config": []map[string]interface{}{
-						{
-							"src_ip_ranges": schema.NewSet(schema.HashString, convertStringArrToInterface(rule.Match.Config.SrcIpRanges)),
-						},
-					},
+					"config":         flattenMatchConfig(rule.Match.Config),
```

Comment:
Even though it looks here like Match is always set, we've been burned before by nil pointers after APIs have changed. Can you also add a nil check?